src/AppBundle/Resources/config/services.yml

```diff
@@ -11,3 +11,13 @@ services:
         arguments: ['@session', '@router']
         tags:
             - { name: kernel.event_listener, event: kernel.request , method: preCreate }
+
+    sylius_demo.listener.user_registration:
+        class: AppBundle\EventListener\UserRegistrationListener
+        arguments: ['@session', '@sylius.admin_user.token_generator.email_verification', '@sylius_demo.generator.verification_flash_message']
+        tags:
+            - { name: kernel.event_listener, event: sylius.customer.post_register , method: addVerificationLink }
```

Comment:
We don't need to replace whole `UserRegistrationListener`, there is `REQUEST_VERIFICATION_TOKEN` event dispatched after token generation. On that event we should add flash with the token.